tools/proguard5.3.2/test/proguard/util/ArrayUtilTest.java

```diff
@@ -0,0 +1,614 @@
+package proguard.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ArrayUtilTest {
+
+	@Test
+	public void testEqualByte() {
+		Assert.assertFalse(ArrayUtil.equal(
+				new byte[]{3, 2, 1}, new byte[]{1, 2, 3}, 4));
```

Comment:
I'd put the first array on the above line to make it more standard with the other asserts.